content/en/ddsql_editor/reference_tables.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reference Tables in DDSQL
+further_reading:
+- link: "/integrations/guide/reference-tables/"
+  tag: "Documentation"
+  text: "Add Custom Metadata with Reference Tables"
+---
+
+# Overview
+
+Reference tables are an essential feature within DDSQL, designed to help you organize and simplify your data queries. These tables store predefined sets of information that can be easily referenced within your queries, reducing complexity and enhancing query performance. In addition to querying data, you can join reference tables with other datasets, expanding your analytical capabilities and insights.
+
+For more information on adding reference tables, see the [Reference Tables Documentation](1).
+
+## Querying Reference Tables
+
+You can query reference tables directly with the DDSQL Editor. This guide aims to clarify how you can unlock the full potential of reference tables in your data queries.
+
+### Example Query Syntax
+
+To query a reference table, you can use the following syntax. Assume the reference table is named "test":
+
+```sql
+SELECT * FROM reference_tables.test
+```
+
+This query retrieves all the data from the specified reference table. Modify the query to include specific columns or conditions as needed.
+
+### Joining Data
+
+In addition to querying reference tables, you can also join them with other available datasets. This process allows for richer data analysis and insights. By joining reference tables, you can:
```

Comment:
Let's avoid using the word "dataset" as this might become a specific concept within datadog referring to something different. 
Let's use "table"